nanofi/tests/CTestsBase.h

```diff
@@ -138,7 +138,8 @@ class TestControllerWithTemporaryWorkingDirectory {
   }
 
   ~TestControllerWithTemporaryWorkingDirectory() {
-    chdir(old_cwd_);
+    int success = chdir(old_cwd_);
+    (void) success;
```

Comment:
is this added to silence an "unused return value" warning?

Reply:
yes